code/lib/test/README.md

```diff
@@ -0,0 +1,38 @@
+# Storybook Test
+
+The `@storybook/test` package contains utilities to be used to test your story in the `play` function.
+
+## Installation
+
+Install this addon by adding the `@storybook/test` dependency:
+
+```sh
+npm install -D @storybook/test
+pnpm add -D @storybook/test
+yarn add -D @storybook/test
+```
+
+## Usage
+
+The test package exports an instrumented version `@vitest/spy`, `@vitest/expect` (based on `chai`), `@testing-library/dom` and `@testing-library/user-event`.
```

Comment:
Might be nice to link to the other packages, in case someone wants to learn more about them and how they work.

```suggestion
The test package exports instrumented versions of [@vitest/spy](https://vitest.dev/api/mock.html), [@vitest/expect](https://vitest.dev/api/expect.html) (based on [chai](https://www.chaijs.com/)), [@testing-library/dom](https://testing-library.com/docs/dom-testing-library/intro) and [@testing-library/user-event](https://testing-library.com/docs/user-event/intro).
```